### ISSUE TITLE
Why the output of the ensemble model processing with softmax and log before evaluating?

### ISSUE THREAD

**Issue by HuangZuShu** [AI-secure/Transferability-Reduced-Smooth-Ensemble#6]
Hi @Lucas110550 , 
After a thorough review of your code, i find the [ensemble.py](https://github.com/AI-secure/Transferability-Reduced-Smooth-Ensemble/blob/main/models/ensemble.py) is mainly used for evaluating robustness.
But i don't know why the output of the ensemble model processing with softmax and log before evaluating ?
Also, `nn.CrossEntropyLoss` in pytorch has arleady combined  `nn.LogSoftmax` and  `nn.NLLLoss`. So should we remove `F.softmax` and `torch.log` ? 
The final version i want to change and your version as following:

https://github.com/AI-secure/Transferability-Reduced-Smooth-Ensemble/blob/1d246ddef7a7e6ea600da9eea26264a65c0db6f9/models/ensemble.py#L13-L20

```python
 def forward(self, x): 
     if len(self.models) > 1: 
         outputs = 0 
         for model in self.models: 
             outputs += model(x)
         output = outputs / len(self.models) 
         return output
```
